drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -956,6 +956,10 @@ void nrf_802154_receive_failed(nrf_802154_rx_error_t error, uint32_t id)
 		break;
 	}
 
+#if IS_ENABLED(CONFIG_IEEE802154_NRF5_LOG_RX_FAILURES)
```

Comment:
Would be good to use regular `if (IS_ENABLED(CONFIG_IEEE802154_NRF5_LOG_RX_FAILURES)) {`, so that the code is still compiled.